src/xmipp/libraries/reconstruction/shift_particles.cpp

```diff
@@ -109,16 +117,16 @@
 		MDRow rowGeo;
 		rowGeo.setValue(MDL_SHIFT_X, -posp(0));
 		rowGeo.setValue(MDL_SHIFT_Y, -posp(1));
-	    A.initIdentity(3);
-	    geo2TransformationMatrix(rowGeo, A, true);
+		A.initIdentity(3);
+		geo2TransformationMatrix(rowGeo, A, true);
 
-    	I().setXmippOrigin();
-    	Iout().resize(1, 1, boxSize, boxSize, false);
-    	Iout().setXmippOrigin();
+		I().setXmippOrigin();
+		Iout().resize(1, 1, boxSize, boxSize, false);
+		Iout().setXmippOrigin();
 		applyGeometry(LINEAR, Iout(), I(), A, IS_NOT_INV, false, 0.);
 
 		ix_particle++;
-		FileName out = formatString("%d@%s.mrcs", ix_particle, fnOut.c_str());
+		FileName out = formatString("%d@%s", ix_particle, fnOut.c_str());
 		Iout.write(out);
 		mdParticles.setValue(MDL_IMAGE, out, ix_particle);
 		mdParticles.setValue(MDL_SHIFT_X, shiftx+posp(0), ix_particle);
```

Comment:
I don't think that this is correct. I think that the  shift describes 'how much should I move the particle to center it'. Since you have done that above, I would expect the shift to be 0